unrar@7.01: download via aria2 failed

Installing 'unrar' (7.01) [64bit] from 'main' bucket
Starting download with aria2 ...
Download: 05/18 18:45:14 [ERROR] CUID#8 - Download aborted. URI=https://raw.githubusercontent.com/ScoopInstaller/Binary/master/unrar/7.01/UnRAR_x64Download: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://raw.githubusercontent.com/ScoopInstaller/Binary/master/unrar/7.01/UnRAR_x64.eDownload: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 9219f7|ERR |       0B/s|C:/Scoop/cache/unrar#7.01#https_raw.githubusercontent.com_ScoopInstaller_Binary_master_unrar_7.01_UnRAR_x64.exe_UnRAR.exe
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 3) Resource was not found
ERROR https://raw.githubusercontent.com/ScoopInstaller/Binary/master/unrar/7.01/UnRAR_x64.exe#/UnRAR.exe
    referer=https://raw.githubusercontent.com/ScoopInstaller/Binary/master/unrar/7.01/UnRAR_x64.exe#/
    dir=C:\Scoop\cache
    out=unrar#7.01#https_raw.githubusercontent.com_ScoopInstaller_Binary_master_unrar_7.01_UnRAR_x64.exe_UnRAR.exe

ERROR & 'C:\Scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Scoop\cache\unrar.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/7.4 (Windows NT 10.0; Win64; x64; Core)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=14416 --continue --summary-interval=0 --auto-save-interval=1

--- a/bucket/unrar.json
+++ b/bucket/unrar.json
@@ -9,11 +9,11 @@
     "architecture": {
         "64bit": {
             "url": "https://raw.githubusercontent.com/ScoopInstaller/Binary/master/unrar/7.01/UnRAR_x64.exe#/UnRAR.exe",
-            "hash": "90c2b2107a22ea8eb3593a155c4c0007b18b1ba552bf65f963c040038da248be"
+            "hash": "e51021f6cb20efbd2169f2a2da10ce1abca58b4f5f30fbf4bae931e4ecaac99b"
         },
         "32bit": {
             "url": "https://raw.githubusercontent.com/ScoopInstaller/Binary/master/unrar/7.01/UnRAR.exe",
-            "hash": "c9e67b0474d3081827a73f1a7080b735d1d80d2d5f5c4aee508b335ed7e8a05a"
+            "hash": "d009b6d4eda14b441b0bf421f21631d8314f9596d4f2f04a6c926290f5180228"
         }
     },
     "bin": "UnRAR.exe",

